stock_voucher/models/stock_picking.py

```diff
@@ -66,66 +65,60 @@ def get_estimated_number_of_pages(self):
         if lines_per_voucher == 0:
             estimated_number_of_pages = 1
         else:
-            operations = len(self.pack_operation_ids)
+            operations = len(self.move_lines)
             estimated_number_of_pages = int(ceil(
                 float(operations) / float(lines_per_voucher)
             ))
         return estimated_number_of_pages
 
-    @api.one
     @api.constrains('picking_type_id')
     @api.onchange('picking_type_id')
     def _get_book(self):
         self.book_id = self.picking_type_id.book_id
```

Comment:
```python
for rec in self:
    rec.update({'book_id': rec.picking_type_id.book_id})
```

This because we are using it as api.constrains, so need to support multiple items